whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/attrs/InetnumStatus.java

```diff
@@ -107,6 +107,11 @@ public boolean worksWithParentInHierarchy(final InetStatus parentInHierarchyMain
         return true;
     }
 
+    @Override
+    public boolean isAdministrativeResource() {
+        return this.equals(ALLOCATED_UNSPECIFIED);
```

Comment:
This is not precise enough. This status may be used on out-of-region placeholders or the root 0/0 object.

According to https://www.iana.org/assignments/rdap-json-values/rdap-json-values.xhtml
```
The object instance has been allocated administratively (i.e., not for use by the recipient in their own right in operational networks).
```

We need to implement this separately.

Reply:
I removed this, we are not handling status query parameter in this PR
So I don't need to check if it is an administrative Resource or not